src/KeyConnector/HostedServices/TransferToHostedService.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Bit.KeyConnector.Repositories;
+using Bit.KeyConnector.Services;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Bit.KeyConnector.HostedServices
+{
+    public class TransferToHostedService : IHostedService, IDisposable
```

Comment:
⛏️: is it possible to add Unit Tests?

Reply:
This project has no unit test project set up, so I would not know where/how to add those.